src/app/components/room/action.service.js

```diff
@@ -83,6 +84,15 @@
       LogService.add(entry);
```

Comment:
I would go one step further and re-implement `destroyFeed`to look like that:
```javascript
function destroyFeed(feedId) {
       unpublishFeed(feedId);
       // Log the event
       var entry = new LogEntry("destroyFeed", {feed: feed});
       LogService.add(entry);
}
```

That will reduce code duplication.